peps/pep-0815.rst

```diff
@@ -272,19 +279,14 @@ squatting much easier. For example, one could create a malicious
 ``numpy-cuda`` package that users will be lead to believe it’s a CUDA
 variant of NumPy.
 
-.. code:: bash
-
-    pip install xgboost      # NVIDIA GPU variant
-    pip install xgboost-cpu  # CPU-only variant
-
-`cupy <https://github.com/cupy/cupy>`__ for diverse reasons had to build
+For diverse reasons, `CuPy <https://cupy.dev/>`__ had to build
```

Comment:
I'd drop the "For diverse reasons", if we don't list the reasons, it doesn't help to say that such reasons exist, that's already implied by the "had to".